app/views/pursuits/_pursuit.html.haml

```diff
@@ -0,0 +1,13 @@
+.pursuit{ :id => "#{format_url_name(pursuit.name)}" }
+  .row
+    .col.s12.m7
+      .card
+        .card-image
+          %img(src="jet-skiing.jpg")
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))